measures/apps/measuresdata/measuresdata.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+#
+# measuresdata.sh
+# Must be executable. Call it with all defaults or arguments for measuresdata
+# It is a test script to enable writing python (or other) versions
+
+echo Calling measuresdata $*
+measuresdata $*
```

Comment:
I don't know what this tool does, but this is an unusual way to call an application. Normally you use "$@" which passes on the arguments as if it where called directly.
I think $* will just lump them into one big string which might cause issues with whitespace and quoting.